manifests/mlops-sagemaker/sagemaker-templates-modules.yaml

```diff
@@ -1,7 +1,16 @@
 name: service-catalog
-path: git::https://github.com/awslabs/aiops-modules.git//modules/sagemaker/sagemaker-templates-service-catalog?ref=release/1.5.0&depth=1
+# path: git::https://github.com/awslabs/aiops-modules.git//modules/sagemaker/sagemaker-templates-service-catalog?ref=release/1.5.0&depth=1
+path: modules/sagemaker/sagemaker-templates-service-catalog
 targetAccount: primary
 parameters:
+  - name: repository_type
+    value: GitHub
+  - name: repository_owner
+    value: <github_account_owner>
+  - name: repository_access_token
```

Comment:
This refers to the name of the secret that contains the access token, correct? Should we correct the name of the parameter please

Reply:
Yes, I will change this parameter name